server/routes_create_test.go

```diff
@@ -21,9 +22,14 @@ import (
 
 var stream bool = false
 
-func createBinFile(t *testing.T, kv map[string]any, ti []llm.Tensor) string {
+func createBinFile(t *testing.T, kv map[string]any, ti []llm.Tensor) (string, string) {
 	t.Helper()
 
+	modelDir := os.Getenv("OLLAMA_MODELS")
```

Comment:
Shouldn't this be set to a temp directory as part of the test, to not interfere with a running instance?